Sources/Automerge/Document.swift

```diff
@@ -768,6 +768,61 @@ public final class Document: @unchecked Sendable {
         }
     }
 
+    /// Get the list of marks for a text object at ``Cursor``.
+    ///
+    /// Use:
+    /// ```
+    /// let doc = Document()
+    /// let textId = try doc.putObject(obj: ObjId.ROOT, key: "text", ty: .Text)
+    ///
+    /// let cursor = try doc.cursor(obj: textId, position: 0)
+    /// let marks = try doc.marksAt(obj: textId, cursor: cursor)
+    /// ```
+    ///
+    /// - Parameters:
+    ///   - obj: The identifier of the text object.
+    ///   - cursor: The cursor that indicates a position within the text.
+    ///   - heads: The set of ``ChangeHash`` that represents a point of time in the history the document.
```

Comment:
Since this is set up as optional, include some detail about what the default operation does when the value is not provided by the developer (e.g. "uses the latest version of the document")

Reply:
It was a bit lazy from my side, better to split the method into two variants.

Reply:
Improved the documentation in 63070af2a54f944c6b831143d0d11c481ba3d27f - introducing extra information of differences between using `Cursor` and `Index.